### PR TITLE
Add getApps

### PR DESCRIPTION
**Pull request by santese**

### CODE REVIEW

File: src/lib/types/api/apps.interfaces.ts [hunk 1]
```diff
@@ -40,3 +40,13 @@ export interface App {
 
     dataProperties?: Record<string, unknown>
 }
+
+export interface PliancyApp {
```
Comment (CalebAlbers) [inline src/lib/types/api/apps.interfaces.ts:44]: Our other types don't include a `Pliancy` prefix, so perhaps we can change to just using `App` for consistency

```suggestion
export interface App {
```

**Review by CalebAlbers — APPROVED** [review]
Thanks for adding it!

**Comment by CalebAlbers** [comment]
Thanks for adding a test!

File: src/lib/types/api/apps.interfaces.ts (hunk 1 above)
Comment (santese) [reply src/lib/types/api/apps.interfaces.ts]: @CalebAlbers , there was already an interface named App for the customer but just renamed that to CustomerApp and this to App:
https://github.com/pliancy/api-sdk/blob/9af1a737ab2d01983e9395f753ff4791ae1b558b/src/lib/types/api/apps.interfaces.ts#L26-L52

Comment (CalebAlbers) [reply src/lib/types/api/apps.interfaces.ts]: That's a good call

**Review by CalebAlbers — APPROVED** [review]

**Pull request merged**